src/Twig/Extension/StatusRuntime.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Twig\Extension;
+
+use Sonata\CoreBundle\Component\Status\StatusClassRendererInterface;
+
+/**
+ * @author Hugo Briand <briand@ekino.com>
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ */
+class StatusRuntime
+{
+    /**
+     * @var StatusClassRendererInterface[]
+     */
+    protected $statusServices = [];
+
+    /**
+     * Adds a renderer to the status services list.
+     *
+     * @param StatusClassRendererInterface $renderer
+     */
+    public function addStatusService(StatusClassRendererInterface $renderer)
+    {
+        $this->statusServices[] = $renderer;
+    }
+
+    /**
+     * @param mixed  $object
```

Comment:
`object $object`, or is it really mixed?

Reply:
I don't know, I copy-pasted the code from the extension.

Reply:
Ok,  please keep it that way then :)